gendo.py

```diff
@@ -3,6 +3,7 @@
 import os
```

Comment:
I support separating stdlib imports from dependency imports and imports from this repo. Like this:
```python
from datetime import date, datetime, timedelta
import os

from pyfieri.guy_fieri import speak
import requests

from gendo import Gendo
```
@nficano thoughts?